L/libcxxwrap_julia/build_tarballs.jl

```diff
@@ -3,21 +3,41 @@
 using BinaryBuilder, Pkg
 
 name = "libcxxwrap_julia"
-version = v"0.6.6"
+version = v"0.7.0"
+
+const is_yggdrasil = haskey(ENV, "BUILD_BUILDNUMBER")
+git_repo = is_yggdrasil ? "https://github.com/JuliaInterop/libcxxwrap-julia.git" : joinpath(ENV["HOME"], "src/julia/libcxxwrap-julia/")
+unpack_target = is_yggdrasil ? "" : "libcxxwrap-julia"
 
 # Collection of sources required to complete build
 sources = [
-    "https://github.com/JuliaInterop/libcxxwrap-julia.git" =>
-    "c1c54414ed8875f963253c8b2e4c960f92e32908",
+    GitSource(git_repo, "7959f09947f95e67ee094a2ee3fb8b7f9be696f8", unpack_target=unpack_target)
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-wget https://github.com/JuliaPackaging/JuliaBuilder/releases/download/v1.0.0-2/julia-1.0.0-${target}.tar.gz
+
 mkdir julia
 cd julia
-tar xf ../julia-1.0.0-${target}.tar.gz
-Julia_PREFIX=$PWD
+
+case "$target" in
+	arm-linux-gnueabihf)
+        wget https://julialang-s3.julialang.org/bin/linux/armv7l/1.3/julia-1.3.1-linux-armv7l.tar.gz
+        tar xf julia-1.3.1-linux-armv7l.tar.gz
+        Julia_PREFIX=$PWD/julia-1.3.1
+        ;;
+	x86_64-linux-gnu)
+        wget https://julialang-s3.julialang.org/bin/linux/x64/1.3/julia-1.3.1-linux-x86_64.tar.gz
+        tar xf julia-1.3.1-linux-x86_64.tar.gz
+        Julia_PREFIX=$PWD/julia-1.3.1
+        ;;
+    x86_64-apple-darwin14|x86_64-w64-mingw32)
+        wget https://github.com/Gnimuc/JuliaBuilder/releases/download/v1.3.0/julia-1.3.0-${target}.tar.gz
+        tar xf julia-1.3.0-${target}.tar.gz
+        Julia_PREFIX=$PWD/juliabin
+        ;;
```

Comment:
Can we have these tarballs as sources?  You can use the `unpack_target` argument to unpack the archives into a nicer place.  One advantage would be that they would go to the cache, avoiding downloading them multiple times.